tests/integration-tests/Apple/MacCatalyst.RunAppBundle.proj

```diff
@@ -0,0 +1,38 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
seems like these three test projects could be consolidated, not blocking your PR though.

Reply:
@MattGal good point. I found out about this `<AdditionalProperties>` property and used it to create a template `.proj` which I can now call repeatedly. Works like a charm, I even added a 4th work item!